### PR TITLE
[DA-116] Enhance the documentation of the CLI scripts

### PR DESCRIPTION
**Pull request by jomrazek**

### CODE REVIEW

File: cli/readme.md [hunk 1]
```diff
@@ -12,12 +12,13 @@ Download: [listings.sh](https://raw.githubusercontent.com/project-ncl/dependency
...
-* Check all dependencies from pom in working directory and print their Black/White list status
-* Check if an artifact was built and if so, see the available built versions of that artifact
-* Generate report for an artifact
-* Generate reports based on POM with the information about built artifacts and B/W status
+* check : Check if artifact GROUP_ID:ARTIFACT_ID:VERSION is in black or white list
+* list  : List all artifacts in black or white list
+* lookup: Check if an artifact was built and if so, see the available built versions of that artifact
+* report: Generate dependency report for an artifact
+* pom-bw: Check all dependencies from pom in working directory and print their Black/White list status
+* pom-report :Generate reports based on POM with the information about built artifacts, best match versions
```
Comment (jbartece) [inline cli/readme.md]: Remove space before colon, add one after it

File: cli/readme.md [hunk 2]
```diff
@@ -49,8 +50,8 @@ Download: [listigs.sh](https://raw.githubusercontent.com/project-ncl/dependency-
 Distribution for admins who will be also adding and removing artifacts.
 
 Features are the same as for developers plus:
- * Add artifact GROUP_ID:ARTIFACT_ID:VERSION to black or white list
- * Delete artifact GROUP_ID:ARTIFACT_ID:VERSION from black or white list
+* add : Add artifact GROUP_ID:ARTIFACT_ID:VERSION to black or white list
```
Comment (jbartece) [inline cli/readme.md]: s/to/to the

File: cli/readme.md [hunk 3]
```diff
@@ -49,8 +50,8 @@ Download: [listigs.sh](https://raw.githubusercontent.com/project-ncl/dependency-
...
+* delete: Delete artifact GROUP_ID:ARTIFACT_ID:VERSION from black or white list
```
Comment (jbartece) [inline cli/readme.md]: s/to/to the

File: cli/readme.md [hunk 4]
```diff
@@ -12,12 +12,13 @@ Download: [listings.sh](https://raw.githubusercontent.com/project-ncl/dependency
 Distribution for developers who need just to check artifacts.
 
 Features are:
-* Check if artifact GROUP_ID:ARTIFACT_ID:VERSION is in black or white list
-* List all artifacts in black or white list
-* Check all dependencies from pom in working directory and print their Black/White list status
-* Check if an artifact was built and if so, see the available built versions of that artifact
-* Generate report for an artifact
-* Generate reports based on POM with the information about built artifacts and B/W status
+* check : Check if artifact GROUP_ID:ARTIFACT_ID:VERSION is in black or white list
```
Comment (jbartece) [inline cli/readme.md]: an artifact;
in the black

File: cli/readme.md [hunk 5]
```diff
@@ -12,12 +12,13 @@ Download: [listings.sh](https://raw.githubusercontent.com/project-ncl/dependency
...
+* list  : List all artifacts in black or white list
```
Comment (jbartece) [inline cli/readme.md]: Remove space before colon

File: cli/readme.md [hunk 6]
```diff
@@ -12,12 +12,13 @@ Download: [listings.sh](https://raw.githubusercontent.com/project-ncl/dependency
...
+* lookup: Check if an artifact was built and if so, see the available built versions of that artifact
+* report: Generate dependency report for an artifact
+* pom-bw: Check all dependencies from pom in working directory and print their Black/White list status
```
Comment (jbartece) [inline cli/readme.md]: a pom
in the working directory (probably)?

File: cli/readme.md (hunk 3 above)
Comment (jomrazek) [inline cli/readme.md]: unknown command

Comment (janinko) [inline cli/readme.md]: s/from/from the/

**Comment by janinko** [comment]
:+1: please squash

**Comment by jomrazek** [comment]
@janinko squashed

**Pull request merged**